pkgs/development/libraries/kde-frameworks/kde-wrapper.nix

```diff
@@ -61,13 +61,9 @@ stdenv.mkDerivation {
         fi
     done
 
-    ln -s "$env/share" "$out"
```

Comment:
This would give us the same problem -- unwrapped packages' `propagated-user-env-pkgs` would get there too, and so we'll get collisions later.